core/src/main/java/org/apache/calcite/sql/SqlDialect.java

```diff
@@ -749,6 +749,9 @@ public boolean supportsDataType(RelDataType type) {
     return true;
   }
 
+ /** Returns SqlNode for type in "cast(column as type)" , which is different
```

Comment:
Not sure, shall we wrap `cast ...` with `{@code}`?

Reply:
> which is different in every kind of database such as type name, precision etc.

How about:
which might be different between databases by type name, precision etc.

Reply:
> Not sure, shall we wrap `cast ...` with `{@code}`?

Actually, I think it is not easily understand either.